src/status_im/contexts/wallet/events/collectibles.cljs

```diff
@@ -0,0 +1,127 @@
+(ns status-im.contexts.wallet.events.collectibles
```

Comment:
Moved collectible events to a new namespace (mentioned in PR description).

I'm open to suggestions, if you think this is not the path we want to follow, I can revert it :+1: